gnocchiclient/v1/metric.py

```diff
@@ -215,7 +215,7 @@ def get_measures(self, metric, start=None, stop=None, aggregation=None,
         return [(iso8601.parse_date(ts), g, value)
                 for ts, g, value in measures]
 
-    def aggregation(self, metrics, query=None,
+    def aggregation(self, metrics, query=None, filter=None,
```

Comment:
We can have only one param and check if it's a string or a dict ?

Reply:
Good idea, let's do that.